packages/buy-button/src/__tests__/BuyButton.test.js

```diff
@@ -1,16 +1,36 @@
 import React from 'react'
 import { shallow } from 'enzyme'
 
-import BuyButton from '../index'
+import { BuyButton } from '../index'
 
 describe('<BuyButton />', () => {
-  it('should pass', () => {
-    expect(1).toBe(1)
+  let wrapper
+
+  const props = {
+    quantity: 1,
+    skuId: 1,
+    seller: '1',
+    salesChannel: '1',
+    orderFormId: '12321',
+    mutate: jest.fn(),
+    afterClick: jest.fn(),
+  }
+
+  beforeEach(() => {
+    wrapper = shallow(<BuyButton {...props}> teste </BuyButton>)
   })
 
-  it('should be rendered', () => {
-    const wrapper = shallow(<BuyButton />)
+  it('should match snapshot', () => {
+    expect(wrapper).toMatchSnapshot()
+  })
 
+  it('should be rendered', () => {
     expect(wrapper).toBeDefined()
   })
+
+  it('should call afterClick', () => {
+    expect(props.afterClick.mock.calls.length).toBe(0)
+    wrapper.find('Button').simulate('click')
+    expect(props.afterClick.mock.calls.length).toBe(1)
```

Comment:
Should also test if the mutate function was called

Reply:
👍